modules/ROOT/pages/amazon/amazon-dynamodb-connector-examples.adoc

```diff
@@ -0,0 +1,700 @@
++
+The primary key attributes and index key attributes are automatically copied, no matter what value you set for the `<Projection>` element.
++
+* `ProvisionedThroughput` provisions throughput for read and write activity.
+* `KeySchema` specifies the key schema for the index.
+
+[source,xml,linenums]
+----
+<set-variable value='#[[
+            {
+                "IndexName": "CreateDateIndex",
+                "Projection": {
+                    "ProjectionType": "ALL"
+                },
+                "ProvisionedThroughput": {
+                    "WriteCapacityUnits": 5,
+                    "ReadCapacityUnits": 5
+                },
+                "KeySchema": [
+                    {
+                        "KeyType": "HASH",
+                        "AttributeName": "CreateDate"
+                    },
+                    {
+                    	"KeyType": "RANGE",
+                    	"AttributeName": "IssueId"
+                    }
+                ]
+            } as Object {
+            	class: "org.mule.extension.dynamodb.api.model.GlobalSecondaryIndex"
+            },
+            {
+                "IndexName": "TitleIndex",
+                "Projection": {
+                    "ProjectionType": "ALL"
+                },
+                "ProvisionedThroughput": {
+                    "WriteCapacityUnits": 5,
+                    "ReadCapacityUnits": 5
+                },
+                "KeySchema": [
+                    {
+                        "KeyType": "HASH",
+                        "AttributeName": "Title"
+                    },
+                    {
+                    	"KeyType": "RANGE",
+                    	"AttributeName": "IssueId"
+                    }
+                ]
+            } as Object {
+            	class: "org.mule.extension.dynamodb.api.model.GlobalSecondaryIndex"
+            },
+            {
+                "IndexName": "DueDateIndex",
+                "Projection": {
+                    "ProjectionType": "ALL"
+                },
+                "ProvisionedThroughput": {
+                    "WriteCapacityUnits": 5,
+                    "ReadCapacityUnits": 5
+                },
+                "KeySchema": [
+                    {
+                        "KeyType": "HASH",
+                        "AttributeName": "DueDate"
+                    }
+                ]
+            } as Object {
+            	class: "org.mule.extension.dynamodb.api.model.GlobalSecondaryIndex"
+            }
+        ]]' doc:name="Secondary" variableName="secondary"/>
+----
+
+[[configure-logger]]
+=== Configure a Logger Component
+
+Configure a `Logger` component to print to the Mule console the response generated by the `Create Table` operation:
+
+. In the *Mule Palette*, search for `logger`.
+. Drag the *Logger* component to the right of *Create Table* on the Studio canvas.
+. Enter `#[payload]` in the *Message* field of the Logger properties window.
```

Comment:
Format the name of the window as displayed in the UI, or lowercase "logger" if this is not a label.